pkg/portal/graphql/app_mutation.go

```diff
@@ -385,6 +390,7 @@ var _ = registerMutationField(
 		Resolve: func(p graphql.ResolveParams) (interface{}, error) {
 			input := p.Args["input"].(map[string]interface{})
 			appID := input["id"].(string)
+			phoneNumber := input["phoneNumber"].(string)
```

Comment:
I set the custom attribute to be of type Phone Number. This type will ensure the phone number is in E.164 format. But the phone number actually submitted to Admin API is just the national number (that is, `98765432` instead of `+85298765432`).

Reply:
Updated to include dial code :pray: